doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1648,6 +1861,27 @@ OCX-189: When inputting -40 dBm to HCI_OPCODE_VS_SET_CONN_TX_PWR (0x3F6), the ac
   The TX power for a connection cannot be less than -20 dBm.
   Controller still output -20 dBm if the setting is -40 dBm.
 
+.. rst-class:: v2-5-0
```

Comment:
We discovered last minute that OCX-188 has re-occurred, so we need to add v2-5-0 to the rst-class on this one.
We have also created a new OCX ticket (OCX-227), since OCX-188 was fixes, but then it broke again.
Is it possible to write it like this?:
OCX-184/OCX-227: If 0 dBm TX power is selected, the FEM/PA TX/RX pins do not toggle correctly

Reply:
I will add them.

Reply:
Updated. Please check.

Reply:
Updated. Please check.